packages/web/pages/index.tsx

```diff
@@ -1,24 +1,18 @@
+import { LoadingState } from '@metafam/ds';
 import { PageContainer } from 'components/Container';
-import { PlayerList } from 'components/PlayerList';
-import { getPlayers } from 'graphql/getPlayers';
-import { InferGetStaticPropsType } from 'next';
 import React from 'react';
 
-type Props = InferGetStaticPropsType<typeof getStaticProps>;
-
-export const getStaticProps = async () => {
-  const players = await getPlayers();
+export const getStaticProps = () => {
   return {
-    props: {
-      players,
+    redirect: {
```

Comment:
Returning a redirect during page generation raises an error. That was killing your Vercel build, now it is dying b/c Hasura has changed on `master`.

Reply:
Can you show me the log ?
This is on the documentation https://nextjs.org/docs/basic-features/data-fetching

Reply:
Ok just saw this in the nextjs doc:

> Note: Redirecting at build-time is currently not allowed and if the redirects are known at build-time they should be added in next.config.js.

So I don't see when this feature can be supported 😵 

So we have to remove all of redirects ? what should we replace it with then ?

Reply:
If this is raising issues, we could do this like so?
```
import React, { useEffect } from "react";
import Router from 'next/router'

...
useEffect(() => {
   const {pathname} = Router
   if(pathname == '/' ){
       Router.push('/hello-nextjs')
   }
 });
 ```
 ref: https://stackoverflow.com/a/58182678/5513532

Reply:
@dysbulic will rebase, would love to see vercel's logs on this build.
It looks like the errors appear only when you specifically ask a dynamic route to render with getStaticPath, but it redirects. Should not happen here, not dynamic page. 

https://nextjs.org/docs/messages/gsp-redirect-during-prerender

Reply:
using redirects in next.config now.